### ISSUE TITLE
0.49-DEV-StratCon - Cannot initiate base attack if previous attack was a failure

### ISSUE THREAD

**Issue by Kailan7401** [MegaMek/mekhq#2512]
### Environment
StratCon alpha test build
Windows 10
OpenJDK 11.0.10

### Description
If you trigger a base attack mission and lose, the base disappears from the map (I believe the intended functionality is for it to remain there and continue to cause penalties until it's destroyed). If you ignore the battle and advance the day (taking the automatic loss, then the base remains on the map as intended, but you can no longer interact with the hex the base is in or right click and manage scenario.

### Files
Campaign save includes a pending base attack battle for ease of duplication
[Campaign Info.zip](https://github.com/MegaMek/mekhq/files/6304630/Campaign.Info.zip)
[megameklog.txt](https://github.com/MegaMek/mekhq/files/6304663/megameklog.txt)
[mekhqlog.txt](https://github.com/MegaMek/mekhq/files/6304666/mekhqlog.txt)

**Comment by Windchild292** [comment]
NPE in MekHQ log appears to be related

**Comment by NickAragua** [comment]
Woof, this one uncovered a lot of stuff. Main issue is probably that, even though the scenario is called "Hostile Facility", it didn't actually spawn a facility.

**Comment by Thom293** [comment]
I am sorry for the stupid question.  How do I download the dev version of this?  I found the stratcon page, but I dont see a way to download a build?

**Comment by JonhyBurner** [comment]
> 
> 
> I am sorry for the stupid question. How do I download the dev version of this? I found the stratcon page, but I dont see a way to download a build?

https://github.com/MegaMek/mekhq/pull/2144#issuecomment-815406297

**Comment by sixlettervariables** [comment]
Error from log:
```
Exception in thread "AWT-EventQueue-0" java.lang.NullPointerException
	at mekhq.campaign.stratcon.StratconScenario.getInfo(StratconScenario.java:167)
	at mekhq.campaign.stratcon.StratconScenario.getInfo(StratconScenario.java:154)
	at mekhq.gui.stratcon.StratconScenarioWizard.setInstructions(StratconScenarioWizard.java:152)
	at mekhq.gui.stratcon.StratconScenarioWizard.setUI(StratconScenarioWizard.java:105)
	at mekhq.gui.stratcon.StratconScenarioWizard.setCurrentScenario(StratconScenarioWizard.java:90)
	at mekhq.gui.StratconPanel.actionPerformed(StratconPanel.java:650)
	at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1967)
	at java.desktop/javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2308)
	at java.desktop/javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:405)
	at java.desktop/javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:262)
	at java.desktop/javax.swing.AbstractButton.doClick(AbstractButton.java:369)
	at java.desktop/javax.swing.plaf.basic.BasicMenuItemUI.doClick(BasicMenuItemUI.java:1020)
	at java.desktop/javax.swing.plaf.basic.BasicMenuItemUI$Handler.mouseReleased(BasicMenuItemUI.java:1064)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6635)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3342)
	at java.desktop/java.awt.Component.processEvent(Component.java:6400)
	at java.desktop/java.awt.Container.processEvent(Container.java:2263)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5011)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2321)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4843)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4918)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4547)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4488)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2307)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2772)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4843)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:772)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:715)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:95)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:745)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:743)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
```

**Comment by NickAragua** [comment]
This particular behavior should be fixed with 133e5865e2ee8dd5900b26dfe5f9aa0459a1a04c
